client/state/purchases/test/selectors.js

```diff
@@ -88,6 +95,7 @@ describe( 'selectors', () => {
 				includedDomain: undefined,
 				includedDomainPurchaseAmount: undefined,
 				introductoryOffer: null,
+				isAutoRenewEnabled: true,
```

Comment:
Does it make sense to add a test for the other purchase to ensure that `isAutoRenewEnabled` is `false`?